jupyter-docker/Dockerfile

```diff
@@ -215,7 +215,8 @@ RUN R -e 'install.packages(c( \
     "dplyr",  \
     "tidyr",  \
     "reshape2",  \
-    "bigrquery"),  \
+    "bigrquery",  \
+   "tidyverse"), \
```

Comment:
Echoing @vkumra-broad question about whether there is library duplication now that we're installing tidyverse.